File: app/src/main/java/com/kazakago/cleanarchitecture/domain/usecase/AboutUseCase.java
package com.kazakago.cleanarchitecture.domain.usecase;

/**
 * About UseCase
 *
 * @author Kensuke
 */
public interface AboutUseCase {

    String getPlayStoreUrl();

    String getMailUrl();

    String getWebSiteUrl();

    String getCurrentVersion();

    int getCurrentYear();

}

File: app/src/main/java/com/kazakago/cleanarchitecture/domain/usecase/AboutUseCase.kt
package com.kazakago.cleanarchitecture.domain.usecase

/**
 * About UseCase
 *
 * @author Kensuke
 */
interface AboutUseCase {

    val playStoreUrl: String

    val mailUrl: String

    val webSiteUrl: String

    val currentVersion: String

    val currentYear: Int

}
